dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/oracle/task/GroupTask.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2022-2024 Google LLC
+ * Copyright 2013-2021 CompilerWorks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.edwmigration.dumper.application.dumper.connector.oracle.task;
+
+import com.google.edwmigration.dumper.application.dumper.task.Summary;
+import com.google.edwmigration.dumper.application.dumper.task.Task;
+import java.util.Optional;
+import javax.annotation.CheckForNull;
+import javax.annotation.Nonnegative;
+import javax.annotation.ParametersAreNonnullByDefault;
+
+@ParametersAreNonnullByDefault
+public interface GroupTask extends Task<Summary> {
+
+  @CheckForNull
+  public Exception getException();
+
+  @Nonnegative
```

Comment:
What does the `@Nonnegative` annotation mean in this context?

Reply:
It was a typo, I changed it to `@Nonnull`, which is what was intended